app/src/main/AndroidManifest.xml

```diff
@@ -10,15 +10,16 @@
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
     <uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+    
+    
+    <uses-permission android:name="android.permission.MANAGE_USERS" />
 
     <application
```

Comment:
**Reporter**: ANDROIDLINT
**Rule**: GoogleAppIndexingWarning
**Severity**: WARN
**File**: C:/Users/anisart/.jenkins/workspace/test3/app/src/main/AndroidManifest.xml L17

Missing support for Firebase App Indexing&#10;App is not indexable by Google Search; consider adding at least one Activity with an ACTION-VIEW intent filter. See issue explanation for more details.&#10;Adds URLs to get your app into the Google index, to get installs and traffic to your app from Google Search.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1407726762>*